test/e2e/serial/tests/workload_overhead.go

```diff
@@ -147,14 +147,11 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload overhea
 
 				// need a zone with resources for overhead, pod and a little bit more to avoid zone saturation
 				zoneRequiredResources := rtClass.Overhead.PodFixed.DeepCopy()
-				zoneRequiredResources.Cpu().Add(*podResources.Cpu())
-				zoneRequiredResources.Cpu().Add(*minRes.Cpu())
-
-				zoneRequiredResources.Memory().Add(*podResources.Memory())
-				zoneRequiredResources.Memory().Add(*minRes.Memory())
+				resourcelist.AddCoreResources(zoneRequiredResources, *podResources.Cpu(), *podResources.Memory())
+				resourcelist.AddCoreResources(zoneRequiredResources, *minRes.Cpu(), *minRes.Memory())
```

Comment:
adding minRes as is now will result in a total of CPUs with portions, and due to a gap of not accounting for CPU portions in NRT, the consumed resources will not be reflect in NRT. thus need to round this up, either the after all addition (resourcelist.RoundUp) or nested round up in  AddCoreResources.

Reply:
good point! let's however not add unrelated logic to `AddCoreResources` and let's just bump the requirements here.

Reply:
same for the overhead, it is currently 500m cpus. we want to test this is not reflected in the NRT so better that we have 1 cpu

Reply:
another good point! Let's round up the value after the addition.